examples/csi_driver_install_vars.yaml

```diff
@@ -1,18 +1,22 @@
 ---
+ibmcloud_api_key: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"  #IBM Cloud API KEY
```

Comment:
Can this be used from environment variable? I am not comfortable storing keys (admin) on the cluster.